libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -136,19 +145,23 @@ export class TigerWorkspaceDashboards implements IWorkspaceDashboardsService {
             );
         }
 
+        // TODO: filtercontext by identifier
         const filterContextByRef = filterContextRef
             ? await this.getFilterContext(filterContextRef)
             : undefined;
 
+        const filterContextByExportId = options?.exportId
+            ? await this.getFilterContextFromExportId(options.exportId)
+            : undefined;
+
         const dashboard = await this.getDashboardWithSideloads(ref, types);
         const included = dashboard.included || [];
         const insights = included.filter(isVisualizationObjectsItem).map(visualizationObjectsItemToInsight);
         const plugins = included
             .filter(isDashboardPluginsItem)
             .map(convertDashboardPluginWithLinksFromBackend);
-        const filterContext = filterContextByRef
-            ? filterContextByRef
-            : getFilterContextFromIncluded(included);
+        const filterContext =
+            filterContextByExportId || filterContextByRef || getFilterContextFromIncluded(included);
```

Comment:
Can `filterContextByRef`, `filterContextByExportId` be added to a function that calls `getFilterContextFromIncluded` and produces `filterContext` and this function is called from this method and the one above? To keep things more readable and DRY?

Reply:
Actually went one more step and unified also fetching those, see 45e83a763c97b77d19029195293ff7c5d5f7c80b if you consider this fine.